### DISCUSSION TITLE
Status Indicator Toggle from Armory Crate

### DISCUSSION THREAD

**Discussion by arukup** [seerge/g-helper#2303] [Ideas]
Can the ability to turn off status indicator from armory crate lighting settings be added to g helper?

**Comment by seerge** [comment]
@arukup what kind of status indicator are you talking about ?

**Reply by arukup** [reply]
The LED Lights for power on, battery and drive on STRIX SCAR 2024 Models. They can be turned off using armory crate.

**Reply by seerge** [reply]
This is the first time I hear that those can be turned off somehow. Can you post a screenshot of how it looks in AC ?

**Reply by arukup** [reply]
The Setting at the bottom- Status Indicator
![Status](https://github.com/seerge/g-helper/assets/165097235/f72a6d99-81c2-40ba-a69a-fed7011b4f73)

**Reply by seerge** [reply]
@arukup ok, can you quit g-helper, add following line to the beginning of ``%appdata%\GHelper\config.json``

```
"debug": 1,
```

Then open it again, click ``Extra`` and then click a zoom-glass icon there, it will run a quick scan.
After that - change "Status Indicators" in AC to opposite, and click on zoom-glass icon again.

And post a log from 2 scans here ?

**Reply by arukup** [reply]
[scanOFF.txt](https://github.com/seerge/g-helper/files/14765655/scanOFF.txt)
[scanON.txt](https://github.com/seerge/g-helper/files/14765656/scanON.txt)

DISCLOSURE I screwed up and installed keyboard firmware for 2023 model, and since then the left light wont switch off. Other two work well. Looking for solutions and thought you guys can help. :)

**Reply by seerge** [reply]
@arukup ok, can you open ``Extra`` and via DEVS test fields enter following values

``0x000600C2`` and in second field try either ``0`` or ``7`` and try to set them

Let me know if it toggles LEDS on and off

**Reply by arukup** [reply]
Yes, it works. (My left one is bugged still)

**Reply by seerge** [reply]
@arukup ok, great. I will add it as a checkbox in the app then

As for your keyboard, I'm not sure how to help you tbh. Try to check uncheck all backlight settings under ``Extra`` mb ? 

Or try to install firmware for 2024?
If it's broken on firmware level - app won't be able to fix antnyhing.

**Reply by arukup** [reply]
Great. Thank you. Keep up the good work.

**Reply by seerge** [reply]
@arukup ok, this build should allow to you toggle this via ``Extra`` settings
[GHelper.zip](https://github.com/seerge/g-helper/files/14770713/GHelper.zip)

**Reply by arukup** [reply]
I don't see a toggle. Am I missing something?
![Status2](https://github.com/seerge/g-helper/assets/165097235/c82c840f-924f-4092-aabb-883c0c3f8583)

**Reply by seerge** [reply]
@arukup try this build

# [GHelper.zip](https://github.com/seerge/g-helper/files/14772076/GHelper.zip)

**Reply by arukup** [reply]
Still no toggle

**Reply by seerge** [reply]
@arukup i have attached an old build at first :) try to re-download it or just get latest 0.163 from https://github.com/seerge/g-helper/releases
![Screenshot 2024-03-27 115628](https://github.com/seerge/g-helper/assets/5920850/6f276069-a6f9-41d5-89ea-2bacae01f5ae)

**Reply by arukup** [reply]
yup, it works :)

**Comment by TheOnlyOnlyName** [comment]
hmm I don't see this option on my 2024 G14, am I missing something? It says this option was added in the release notes.

**Reply by seerge** [reply]
@arukup only Strix has this option

**Reply by schichulin** [reply]
does Strix 16 2023 have the turn-off option or is it 2024 only?

**Reply by schichulin** [reply]
> does Strix 16 2023 have the turn-off option or is it 2024 only?

answering to myself, only 2024 models have the support

**Comment by schichulin** [comment]
Hi! guys, does the "LED Status Indicators" feature work on Strix 16 2023 models?

**Reply by schichulin** [reply]
> Hi! guys, does the "LED Status Indicators" feature work on Strix 16 2023 models?

https://github.com/seerge/g-helper/discussions/2303#discussioncomment-11523585
